src/style.css

```
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 60px;
+}
+
+.moyo_header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.moyo_nav {
+  display: flex;
+}
+
+.link_nav {
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  margin-left: 20px;
+}
+
+.is-active:first-child {
```

No, need to add it to class where we add this margin
```suggestion
.link_nav:first-child {
```

---

O, thanks, sorry))